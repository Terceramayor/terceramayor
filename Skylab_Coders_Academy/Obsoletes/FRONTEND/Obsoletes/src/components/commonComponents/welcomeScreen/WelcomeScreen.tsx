// eslint-disable-next-line no-use-before-define
import React from 'react';
import { View, Text, Image } from 'react-native';
import welcomeScreenStyles from './welcomeScreenStyles';

export default function WelcomeScreen () {
  const {
    welcomeImageContainer,
    logoImage,
    obsoletesAboutTitle
  } = welcomeScreenStyles;

  return (

        <View style={welcomeImageContainer}>
            <Image source={require('../../../assets/images/logo.png')} style={logoImage}/>
            <Text style={obsoletesAboutTitle}>Obsoletes</Text>
        </View>

  );
}
