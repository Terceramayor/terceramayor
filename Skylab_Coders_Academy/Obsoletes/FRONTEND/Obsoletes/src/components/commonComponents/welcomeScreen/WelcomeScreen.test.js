// eslint-disable-next-line no-use-before-define
import React from 'react';
import WelcomeScreen from './WelcomeScreen';
import { render } from '@testing-library/react-native';

describe('Given the NavBar component', () => {
  beforeEach(() => {

  });

  describe('When it is rendered as top rated', () => {
    test('Then the SnapShot should match the rendered tree', () => {
      const tree = render(<WelcomeScreen/>);
      expect(tree).toMatchSnapshot();
    });
  });
});
