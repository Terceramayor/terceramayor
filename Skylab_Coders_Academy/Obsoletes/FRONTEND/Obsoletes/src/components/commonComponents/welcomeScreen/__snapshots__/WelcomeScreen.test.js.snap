// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given the NavBar component When it is rendered as top rated Then the SnapShot should match the rendered tree 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#23485D",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/logo.png",
      }
    }
    style={
      Object {
        "height": 100,
        "marginBottom": 65,
        "width": 100,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "white",
        "fontFamily": "MontserratRegular",
        "fontSize": 40,
      }
    }
  >
    Obsoletes
  </Text>
</View>
`;
